.env.example

```diff
@@ -0,0 +1,8 @@
+# Full path to the LEGO Universe client
+CLIENT_PATH=/Users/someuser/LEGO Universe
+# Can improve build time
+BUILD_THREADS=1
+# Updates NET_VERSION in CMakeVariables.txt
+BUILD_VERSION=171023
```

Comment:
We probably want to default to ``171022` since most people won't have the DLU client

Reply:
Sure.